lib/commands/install.js

```diff
@@ -44,6 +45,14 @@ class InstallCommand extends Command {
         }, argv, {local})).then(() => this.ui.listr([{
             title: 'Checking for latest Ghost version',
             task: this.version
+        }, {
+            title: 'Fetching release notes',
+            task: releaseNotes.displayReleaseNotes,
+            enabled: () => process.env.GHOST_SKIP_RELEASE_NOTES !== 'true'
```

Comment:
@acburdine I've added this get-out clause so we can skip over this feature in the event GitHub are having issues.